src/Network/Network/ChangeLog.md

```diff
@@ -19,6 +19,9 @@
 --->
 
 ## Upcoming Release
+* Bugfix in Remove peering and connection commandlet for ExpressrouteCircuit scenario
```

Comment:
Unrelated(?)

Reply:
I merged master, I think that's an issue with the merging :/

Reply:
Dont see it in master, remove it

Reply:
done